Formula/percolator.rb

```diff
@@ -0,0 +1,35 @@
+class Percolator < Formula
+  # cite K_ll_2007: "https://doi.org/10.1038/nmeth1113"
+  desc "Semi-supervised learning for peptide identification from shotgun proteomics data"
+  homepage "http://percolator.ms"
+  url "https://github.com/percolator/percolator/archive/rel-3-05.tar.gz"
+  version "3.05"
+  sha256 "5b746bdc0119a40f96bc090e02e27670f91eeb341736911750b170da7e5c06bb"
+  license all_of: ["Apache-2.0", "BSD-2-Clause", "BSD-3-Clause", "MIT"]
+
+  depends_on "boost" => :build
+  depends_on "cmake" => :build
+  depends_on "xerces-c" => :build
+  depends_on "xsd" => :build
+  depends_on "curl"
+
+  def install
+    inreplace "CPack.txt", "set(CMAKE_INSTALL_PREFIX /usr/local)", ""
+
+    if OS.mac?
+      inreplace "CommonCMake.txt", /(?<=if\(UNIX OR APPLE OR MINGW\).)/m,
+                "set(CMAKE_EXE_LINKER_FLAGS='-framework CoreServices -framework CoreFoundation')"
```

Comment:
Is this hack needed, and what's the error message without it? It's helpful to include a comment above that says
`# Fix Error: foobar does not frobnicate`
where the error message is copied verbatim from the failed build, which helps with Googling that error message.

Reply:
This is a WIP -- I don't have a Mac system to test against, but this is targeted against build errors on Mac:

[Example](https://github.com/brewsci/homebrew-bio/runs/1476548379?check_suite_focus=true#step:4:374):

```
Undefined symbols for architecture x86_64:
  "_CFRelease", referenced from:
      xercesc_3_2::MacOSUnicodeConverter::upperCase(char16_t*) in libxerces-c.a(MacOSUnicodeConverter.cpp.o)
      xercesc_3_2::MacOSUnicodeConverter::lowerCase(char16_t*) in libxerces-c.a(MacOSUnicodeConverter.cpp.o)
```